src/components/Reports/Reports.jsx

```diff
@@ -598,26 +598,11 @@ class ReportsPage extends Component {
                   </div>
                   <div>
                   <div className="total-report-item">
-                    {/* <Button color="info" onClick={this.showTotalProject}>
-                      {this.state.showTotalProject ? 'Hide Total Project Report' : 'Show Total Project Report'}
-                    </Button> */}
                     <Button color="info" onClick={this.showTotalTeam}>
-                      {this.state.showTotalTeam
-                        ? 'Hide Total Team Report'
-                        : 'Show Total Team Report'}
+                      {this.state.showTotalTeam ? 'Hide Total Team Report' : 'Show Total Team Report'}
                     </Button>
-                    <div style={{ display: 'inline-block', marginLeft: 10 }}>
-                      <EditableInfoModal
-                        areaName="totalTeamReportInfoPoint"
-                        areaTitle="Total Team Report"
-                        role={userRole}
-                        fontSize={15}
-                        isPermissionPage
-                        darkMode={darkMode}
-                      />
-                    </div>
-                  </div>
-                </div>
+              </div>
+              </div>
                 </div>
```

Comment:
Is the deletion right?

Reply:
I have tested the PR with some changes requested above. Please try and review again.

Reply:
The icon is missing. It's likely due to the accidental deletion of the code snippet while resolving merge conflicts. It's around line 600 in Reports.jsx as my review comment suggests.
![Screenshot_20241012_031507](https://github.com/user-attachments/assets/e0dd9bbb-ead1-4823-bee1-e6940fa32fef)

Reply:
Thanks for pointing that out. I have added the Icon. Please let me know if you find something missing.